packages-presets/preset-wind4/src/utils/handlers/index.ts

```diff
@@ -5,3 +5,5 @@ export const handler = createValueHandler(valueHandlers)
 export const h = handler
 
 export { valueHandlers }
+
+export * from './regex'
```

Comment:
```suggestion
```

I don't think it makes sense to export it for external use. It attacks our internal rules.